src/ui/pages/equipment/asset/Asset.js

```diff
@@ -24,7 +24,7 @@ import NCRIframeContainer from '../../../components/iframes/NCRIframeContainer';
 import useEntity from "hooks/useEntity";
 import { isClosedEquipment, assetLayoutPropertiesMap } from '../EquipmentTools';
 import ClearAllIcon from '@mui/icons-material/ClearAll';
-import { AssetIcon, PartIcon } from 'eam-components/dist/ui/components/icons';
+import { AssetIcon, PartIcon, PartPlusIcon } from 'eam-components/dist/ui/components/icons';
```

Comment:
Why do you import PartPlusIcon

Reply:
I actually don't. It is just like that in my branch for some reason